libshaderc/src/shaderc_cpp_test.cc

```diff
@@ -458,21 +458,29 @@ TEST_F(CppInterface, ForcedVersionProfileRedundantProfileStd) {
 
 TEST_F(CppInterface, GenerateDebugInfoBinary) {
   options_.SetGenerateDebugInfo();
-  // The output binary should contain the name of the vector: debug_info_sample
-  // as char array.
-  EXPECT_THAT(CompilationOutput(kMinimalDebugInfoShader,
-                                shaderc_glsl_vertex_shader, options_),
-              HasSubstr("debug_info_sample"));
+  const std::string binary_output =
+      CompilationOutput(kMinimalDebugInfoShader,
+                        shaderc_glsl_vertex_shader, options_);
+  // The binary output should contain the name of the vector (debug_info_sample)
+  // null-terminated, as well as the whole original source.
+  std::string vector_name("debug_info_sample");
```

Comment:
Using an escaped null character would have been clearer, e.g.   "debug_info_sample\0"

Reply:
Just doing `std::string vector_name("debug_info_sample\0")` wouldn't end up including that null terminator in the comparison since the string constructor would just treat that as normal end of string (if it's not clear - here I'm trying to specifically catch the OpName for debug_info_sample as the test would have previously done, rather than just possibly matching what's in the OpSource, which could happen without checking for the null terminator).